calibration/METBKA/CGINS-METLGR-99991__20150509.csv

```diff
@@ -1,9 +1,9 @@
 serial,name,value,notes
-UNKNOWN,CC_depth_of_conductivity_and_temperature_measurements_m,1.11,CP01CNSM-00003
+UNKNOWN,CC_depth_of_conductivity_and_temperature_measurements_m,0.98,CP03ISSM-00002 CP04OSSM-00002
 UNKNOWN,CC_height_of_air_humidity_measurement_m,4.255,
 UNKNOWN,CC_height_of_air_temperature_measurement_m,4.255,
 UNKNOWN,CC_height_of_windspeed_sensor_above_sealevel_m,4.74,
 UNKNOWN,CC_jcool,1,Constant
 UNKNOWN,CC_jwarm,1,Constant
 UNKNOWN,CC_zinvpbl,600,Constant
-UNKNOWN,CC_use_velpt,0,
+UNKNOWN,CC_use_velpt,1,
```

Comment:
This is correct, but looks confusing.
Essentially CGINS-METLGR-99991__20150507 and CGINS-METLGR-99992__20150507 are deleted (go away). 
A new file is created called CGINS-METLGR-99991__20150509, which was essentially the CGINS-METLGR-99991__20150507 file with a new date and only applying to ISSM and OSSM.